core/internal/gethwrappers/generated/flags_wrapper/flags_wrapper.go

```diff
@@ -1537,6 +1538,36 @@ func (_Flags *Flags) UnpackLog(out interface{}, event string, log types.Log) err
 	return _Flags.FlagsFilterer.contract.UnpackLog(out, event, log)
 }
 
+func (_Flags *Flags) ParseLog(log types.Log) (interface{}, error) {
+	abi, err := abi.JSON(strings.NewReader(FlagsABI))
```

Comment:
can we just export the abi from every contract as part of our wrapper extender? We do this `abi.JSON(strings.NewReader(contractABI)` plenty of times to warrant it.

Reply:
Sure, makes sense. I think I'll do it in a follow-up PR